### DISCUSSION TITLE
three dataset interface  mean from openapi

### DISCUSSION THREAD

**Discussion by qingguox** [ml-for-good/automl#115] [Q&A]
i has some question from three dataset interface.

![Screen Shot 2022-10-17 at 19 21 13](https://user-images.githubusercontent.com/48853704/196164588-86009fc4-26cc-46fb-aa76-9c2f462607d2.png)

what do these three interface mean? 
Are there specific case? 
@free-y

**Comment by free-y** [comment]
It depends on the final UI design, there maybe some pages which will show details of dataset table and columns as well as different dimensions of model evaluations as per dataset annotations. But currently, I think you can ignore those entities.

**Reply by qingguox** [reply]
ok,  so,  I think that they are  actually  a data template of dataset.
